README.md

```diff
@@ -1,70 +1,50 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/MathMagicians-orange)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Math Magicians
 
-## Available Scripts
+"Math magicians" is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to make simple calculations. Read a random math-related quote.
 
-In the project directory, you can run:
+![screenshot](./leaderboard.png)
```

Comment:
- [ ] [Optional] This image doesn't appear so kindly make it appear or delete it to make your readme look professional.